bridge/engine/Engine.h

```diff
@@ -50,6 +55,7 @@ class Engine
     uint32_t _tickCounter;
 
     concurrency::MpmcQueue<utils::Function> _threadQueue;
```

Comment:
If this is a JobManager instead, we can let the Engine thread pick jobs from it and we can still post Callable jobs on it that allows direct method invocation.